docs/user_guide/annotation_tool.md

```diff
@@ -61,12 +61,12 @@ A good way to manage different node versions is to use the [node version manager
 <!--
 NOTE: Raw HTML is used for this set of images as in the README to restrict screenshot size, but mkdocs doesn't play well with the relative path links (see https://github.com/mkdocs/mkdocs/issues/991), so the paths have been modified to start with `../` instead of `./` so mkdocs can find the right files.
 -->
-<img width="300" src="../imgs/annotate/readme_navbar.png" alt="Annotation tool navbar screenshot">
-<img width="300" src="../imgs/annotate/readme_nextpage_button.png" alt="Annotation tool next page button screenshot">
+<img width="300" src="../../imgs/annotate/readme_navbar.png" alt="Annotation tool navbar screenshot">
+<img width="300" src="../../imgs/annotate/readme_nextpage_button.png" alt="Annotation tool next page button screenshot">
```

Comment:
```suggestion
![Annotation tool navbar screenshot](../imgs/annotate/readme_navbar.png){ width="300"}
![Annotation tool next page button screenshot](../imgs/annotate/readme_nextpage_button.png){ width="300"}
```

Reply:
We have the `width` attribute from https://squidfunk.github.io/mkdocs-material/reference/images/ that lets us just go with vanilla markdown syntax here. That should be better and also let the link checker catch the problem in the future